src/app/modules/bounce/bounce.controller.ts

```diff
@@ -13,10 +13,11 @@ const logger = createLoggerWithLabel(module)
  * @param req Express request object
  * @param res - Express response object
  */
-const handleSns = async (
-  req: Request<{}, {}, ISnsNotification>,
-  res: Response,
-) => {
+export const handleSns: RequestHandler<
+  Record<string, string>,
+  string,
```

Comment:
should this be `never` since we shouldn't send anything back to AWS?

Reply:
You are absolutely correct!